// Copyright 2019 SWIR authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";
import "client_api.proto";

option java_multiple_files = true;
option java_package = "rs.swir.api.client";
option java_outer_classname = "SwirClientApiProto";

package swir_public;

service NotificationApi {
    rpc subscriptionNotification (SubscribeNotification) returns (SubscribeResponse) {
    }
}